_posts/components/2018-10-01-autocomplete.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  Autocomplete
+category: components
+description: Whenever a field have the ability of autocomplete its value we should add a hint in form of placeholder and 
```

Comment:
Whenever a field **has** the ability **to** autocomplete its value<b>, (new comma)</b> we should add (...)